src/test/unit/engine/NetworkTest.cpp

```diff
@@ -34,35 +34,21 @@
 
 using namespace nupic;
 
-#define SHOULDFAIL_WITH_MESSAGE(statement, message)                            \
-  {                                                                            \
-    bool caughtException = false;                                              \
-    try {                                                                      \
-      statement;                                                               \
-    } catch (nupic::LoggingException & e) {                                    \
-      caughtException = true;                                                  \
-      EXPECT_STREQ(message, e.getMessage())                                    \
-          << "statement '" #statement "' should fail with message \""          \
-          << message << "\", but failed with message \"" << e.getMessage()     \
-          << "\"";                                                             \
-    } catch (...) {                                                            \
-      FAIL() << "statement '" #statement                                       \
-                "' did not generate a logging exception";                      \
-    }                                                                          \
-    EXPECT_EQ(true, caughtException)                                           \
-        << "statement '" #statement "' should fail";                           \
-  }
+static bool verbose = false;
+#define VERBOSE if(verbose) std::cerr << "[          ]"
+
 
 TEST(NetworkTest, AutoInitialization) {
 
   // Uninitialize NuPIC since this test checks auto-initialization
   // If shutdown fails, there is probably a problem with another test which
   // is not cleaning up its networks.
+  /**
```

Comment:
is this intended to be left out?

Reply:
Oh, good catch.
I was chasing down a access violation and forgot to uncomment the parts that I took out to isolate the problem.

Reply:
yea, it's safer to comment the whole block with //. as those are then more visible in the diff in these cases.